fenix/app/src/main/res/layout/search_selector.xml

```diff
@@ -41,7 +41,7 @@
                 android:layout_height="6dp"
                 android:importantForAccessibility="no"
                 app:srcCompat="@drawable/ic_chevron_down_6"
-                app:tint="@color/photonBlack"
+                app:tint="?attr/textPrimary"
```

Comment:
One correction here - this should be `iconPrimary`. I have the small chevron selected and in the "Colors" section, you will see Light/**Icon**/Primary. `textPrimary` should be used for text as the name implies, but **icon** primary is what you are looking for here.

<img width="608" alt="Screen Shot 2023-02-22 at 5 04 16 PM" src="https://user-images.githubusercontent.com/1190888/220770331-13e79def-af5a-4e07-b714-7f8dd685891d.png">

Reply:
currently there is no iconPrimary attribute, and the majority is the icons in the xml are using android:fillColor="?attr/textPrimary" as a filler.

I will merge it as it is and file a bug to add the attribute and use it